activiti-core/activiti-engine/src/test/resources/META-INF/persistence.xml

```diff
@@ -34,7 +34,7 @@
     <exclude-unlisted-classes>true</exclude-unlisted-classes>
     <properties>
         <property name="jakarta.persistence.jdbc.driver" value="org.h2.Driver" />
-        <property name="jakarta.persistence.jdbc.url" value="jdbc:h2:mem:activiti-jpa;DB_CLOSE_DELAY=1000" />
+        <property name="jakarta.persistence.jdbc.url" value="jdbc:h2:mem:activiti-jpa;DB_CLOSE_DELAY=1000;NON_KEYWORDS=VALUE" />
```

Comment:
Why is this needed? Which use cases is it covering? The [documentation](https://www.h2database.com/html/commands.html#set_non_keywords) suggests to use quoted identifiers if possible.

Reply:
Tried to quote column 'value' in FieldAccessJPAEntity but it does not help

Reply:
Ok, thank you for trying it. Let's keep with `NON_KEYWORDS` in this case.